gpkit/_cvxopt.py

```diff
@@ -2,10 +2,8 @@
 from cvxopt import solvers, spmatrix, matrix, log
 
 
-def cvxoptimize_fn(options=None):
-    "Return a cvxopt solve function for a particular set of options."
-    if options:
-        solvers.options.update(options)
```

Comment:
this update of `solvers.options` has non-intuitive side effects, so I've removed it. For example, if one runs

`m.solve(solver="cvxopt", options={"feastol": 1e-6})`

and then runs

`m.solve(solver="cvxopt")`,

the second call will still have the feastol option from the earlier call.